equinix/resource_network_device.go

```diff
@@ -1106,7 +1144,7 @@ func resourceNetworkDeviceUpdate(ctx context.Context, d *schema.ResourceData, m
 	supportedChanges := []string{
 		neDeviceSchemaNames["Name"], neDeviceSchemaNames["TermLength"], neDeviceSchemaNames["CoreCount"],
 		neDeviceSchemaNames["Notifications"], neDeviceSchemaNames["AdditionalBandwidth"],
-		neDeviceSchemaNames["ACLTemplateUUID"], neDeviceSchemaNames["MgmtAclTemplateUuid"],
+		neDeviceSchemaNames["ACLTemplateUUID"], neDeviceSchemaNames["MgmtAclTemplateUuid"], neDeviceSchemaNames["Secondary"],
```

Comment:
We can create a separate local variable used for secondary object change detection.